pages/code-of-conduct.tsx

```diff
@@ -45,18 +45,21 @@ export default withPageMetadata((props: WithPageMetadataProps) => (
 
     <p>
       If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. Conference
-      staff can be identified by event branded t-shirts and organiser name tags. Harassment and other Code of Conduct
-      violations reduce the value of our event for everyone. We want you to be happy at our event. People like you make
-      our event a better place. You can make a report either personally or anonymously.
+      staff can be identified by {props.pageMetadata.conference.Organiser.BrandColour} event branded t-shirts and organiser
+      name tags. Harassment and other Code of Conduct violations reduce the value of our event for everyone. We want you to
+      be happy at our event. People like you make our event a better place. You can make a report either personally or anonymously.
     </p>
 
     <h3>Personal report</h3>
 
     <p>You can make a personal report by:</p>
 
     <ul>
-      <li>Contacting a staff member, identified by event branded t-shirts and organiser name tags</li>
-      <li>Sending a direct message to our Twitter account: {props.pageMetadata.conference.Socials.Twitter}</li>
+      <li>
+        Contacting a staff member, identified by {props.pageMetadata.conference.Organiser.BrandColour} event branded
```

Comment:
Should we say that if you can't find one to go to our booth in the main foyer that will always have someone present?

Reply:
Interesting. I want to refrain from putting venue specific content in the CoC as that will increase the friction for other conferences to adopt this site. I can put something in around heading to the main DDD conference booth. We can cover location etc on the day?